### ISSUE TITLE
Issue with different sizes of reorderable items and scale

### ISSUE THREAD

**Issue by RicardasBivainis** [Calvin-LL/Reorderable#46]
When dragging large reorderable items in LazyColumn on a device with increased scale or zoom, they sometimes automatically release the drag and trigger the onDragStopped event.
It's quite easy to replicate using demo app, increasing scale of the device and adding bigger items to itemList
```

val sizes = listOf(50, 200, 250,300)

val items = (0..200).map {
    Item(id = it, text = "Item #$it", size = sizes[it % sizes.size])
```

**Comment by Calvin-LL** [comment]
Does it happen with very big items without scaling? Might just be a duplicate of https://github.com/Calvin-LL/Reorderable/issues/4#issuecomment-2107410640. It will be fixed when Compose Foundation v1.7.0 is out on multiplatform. Try using [v2.4.0-alpha02](https://github.com/Calvin-LL/Reorderable/releases/tag/v2.4.0-alpha02).

**Comment by RicardasBivainis** [comment]
It's seems the main problem is with item size, because it's easy to reproduce if you make it something like: 
`val sizes = listOf(50, 300, 450,500)`
2.4.0-alpha02 version does not resolve the issue
